schemas/vulnerability.yml

```diff
@@ -0,0 +1,159 @@
+---
+- name: vulnerability
+  title: Vulnerability
+  group: 2
+  short: Fields to describe the vulnerabilty relevant to an event.
+  description: >
+    The vulnerability fields describe information about a vulnerabilty that is
+    relevant to an event.
+
+  type: group
+  fields:
+
+    - name: classification
+      level: extended
+      type: keyword
+      short: Classification of the vulnerability.
+      description: >
+        The classification of the vulnerability scoring system.
+        For example (https://www.first.org/cvss/)
+
+      example: CVSS
+
+    - name: enumeration
+      level: extended
+      type: keyword
+      short: Identifier of the vulnerability.
+      description: >
+        The type of identifier used for this vulnerability.
+        For example (https://cve.mitre.org/about/)
+
+      example: CVE
+
+    - name: reference
+      level: extended
+      type: keyword
+      short: Reference of the vulnerability.
+      description: >
+        A resource that provides additional information, context, and
+        mitigations for the identified vulnerabiltiy.
+        For example (https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=ssh)
```

Comment:
I would remove this line. I think the `example:` attribute below is sufficient.

Reply:
Done.